### PR TITLE
Hello-rails-Front-end

### PR DESCRIPTION
**Pull request by umair-rao**
Instructions to run the project :
git checkout setup-react
npm install
npm start
This is Front-end of the application In which i have implemented the following things :

Create a new React app called 'hello-react-front-end'
Initialize your project with Git.
Create the App component with react-router.
Create the Greeting component that will display a greeting. Set it up as a route in your App component.
Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
Display the random greeting in your Greeting component.

### CODE REVIEW

**Comment by umair-rao** [comment]
Please Approve my Project as the deadline is near. I have only this activity pending in my dashboard for the final Capstone Project Module.

File: .babelrc
Comment (Efiamotu-1) [inline .babelrc:1]: - [ ] Your project currently doesn't follow all of the project requirements for this milestone, please kindly work on it properly and make the necessary changes :+1: 
- [ ] Kindly edit your readme to also reflect the appropriate project which you are working on :+1: :sparkles: 
prq

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @umair-rao, 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi @umair-rao 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ ❌ 

- Kindly add a customized readme in your backend repo and add a link to the frontend part.

- Please make sure that your project has a Postgres database set up.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @umair-rao 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implement a connection between a Ruby on Rails back-end and React front-end. ✔️ 

## Required Changes ♻️

- [ ] Kindly don't disable any linters in `component/Greetings.js` file, I think you can address the issue that the linter is pointing at. 🙏 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by umair-rao** [comment]
I have **already installed** the mentioned package (**You can verify** from paackage.json file) but the error still prisits therefore I have disabled linters for just this one line. I would Request the reviewer to please mention the solution before rejecting PR since I found the only way.
Screenshot (5)

 
✅✅ **The project is working fine.** ✅✅

**Comment by OyePriscilla** [comment]
Hi @umair-rao 👍, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that ought to be done in a fresh PR; kindly open a new PR and ensure that all the project milestones are done. 🙏 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**